packages/electrode-archetype-react-app/lib/logger.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const archetype = require("../config/archetype");
+const devRequire = archetype.devRequire;
+
+if (devRequire) {
+  const winston = devRequire("winston");
+  const logger = new (winston.Logger)({
+    exceptionHandlers: [
+      new (winston.transports.File)({
+        filename: "archetype-exceptions.log"
+      })
+    ],
+    transports: [
+      new (winston.transports.Console)({
+        level: "info",
+        colorize: true,
+        prettyPrint: true
+      }),
+      new (winston.transports.File)({
+        name: "archetype-loggers-file",
+        filename: "archetype-loggers.log",
+        level: "debug"
+      })
+    ]
+  });
+  module.exports = logger;
+} else {
+  console.verbose = function (message) {
```

Comment:
Updated, below is the output sample:
![screen shot 2017-03-13 at 10 35 58 am](https://cloud.githubusercontent.com/assets/10135897/23867348/e20cb6d2-07d8-11e7-9552-3c5378150a42.png)